docs/menus/pause.json

```diff
@@ -100,39 +121,30 @@
       "station": "Toppings"
     },
     {
-      "label": "M&Ms",
-      "station": "Shake Mixins"
-    },
-    {
-      "label": "Oreo",
-      "station": "Shake Mixins"
-    },
-    {
-      "label": "Reese’s peanut butter cups",
-      "station": "Shake Mixins"
-    },
-    {
       "label": "Cheese Quesadilla (Half / Whole)",
       "special": true,
       "station": "Quesadilla"
     },
     {
+      "description": "Includes tomatoes, onions, green peppers, and jalapeños",
       "label": "Ultimate Quesadilla (Half / Whole)",
       "special": true,
       "station": "Quesadilla"
     },
     {
+      "description": "Pizza built on pita bread; includes two toppings",
       "label": "Pita Pizza",
       "special": true,
       "station": "Pita"
     },
     {
+      "description": "Pizza built on pita bread; includes two toppings",
```

Comment:
I do not think "specialty" items include an "extra two toppings". We need to check with a kitchen manager or worker to be sure.